includes/api/v2/class-wc-rest-orders-v2-controller.php

```diff
@@ -846,7 +846,7 @@ public function get_item_schema() {
 					'context'     => array( 'view', 'edit' ),
 					'readonly'    => true,
 				),
-				'parent_id'            => array(
+				'parent'               => array(
```

Comment:
We can't change the schema for any current endpoint.

Reply:
I've changed it because in `WC_REST_CRUD_Controller::prepare_objects_query` `parent` is used not `parent_id`. Do you have any suggestion what should I do then?

Reply:
You need to fix everything else, but keep the same schema.